lib/Email.js

```diff
@@ -3,7 +3,9 @@
  * For the full license information, please see the LICENSE file distributed with this package.
  */
 
-var _ = require('lodash');
+var cloneDeep = require('lodash/cloneDeep');
```

Comment:
Don't use `var` at all, use `let` and `const`. Prefer `const` over `let`.

Reply:
That's something on my end that should be addressed.